packages/website/docs/getAlgoliaFacets.mdx

```diff
@@ -0,0 +1,153 @@
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@algolia/autocomplete-preset-algolia@alpha"></script>
+<script>
+  const { getAlgoliaFacets } = window['@algolia/autocomplete-preset-algolia'];
+</script>
+```
+
+## Example
+
+```js
+import algoliasearch from 'algoliasearch/lite';
+import { createAutocomplete } from '@algolia/autocomplete-core';
+import { getAlgoliaResults } from '@algolia/autocomplete-preset-algolia';
+
+const searchClient = algoliasearch(
+  'latency',
+  '6be0576ff61c053d5f9a3225e2a90f76'
+);
+
+const autocomplete = createAutocomplete({
+  // ...
+  getSources() {
+    return [
+      {
+        sourceId: 'categories',
+        getItems({ query }) {
+          getAlgoliaFacets({
+            searchClient,
+            queries: [
+              {
+                indexName: 'instant_search',
+                type: 'facet',
+                facet: 'categories',
+                params: {
+                  facetQuery: query,
+                  maxFacetHits: 10,
+                },
+              },
+            ],
+          });
+        },
+        // ...
+      },
+    ];
+  },
+});
+```
+
+:::info
+
+When using `getAlgoliaFacets` and [`getAlgoliaResults`](getAlgoliaResults) with the same search client in different sources or plugins, Autocomplete batches all queries into a single network call to Algolia. If you're using the same search client for different sources or plugins, **make sure to use the same instance to leverage the internal cache and batching mechanism.**
+
+:::
+
+## Parameters
+
+### `searchClient`
+
+> `SearchClient` | required
+
+The initialized Algolia search client.
+
+### `queries`
+
+> `FacetQuery[]` | required
+
+The queries to search for, with the following parameters:
+
+#### `indexName`
+
+> `string` | required
+
+The index name.
+
+#### `params`
+
+> [`SearchForFacetValuesQueryParams` & `SearchOptions`](https://www.algolia.com/doc/api-reference/api-methods/search-for-facet-values/#parameters)
+
+Algolia search for facet values parameters.
+
+These are the default parameters. You can leave them as is and specify other parameters, or override them.
+
+:::info
+
+If you override `highlightPreTag` and `highlightPostTag`, you won't be able to use the built-in highlighting components such as `Highlight`.
+
+:::
+
+```json
+{
+  "highlightPreTag": "__aa-highlight__",
+  "highlightPostTag": "__/aa-highlight__"
+}
+```
+
+### `transformResponse`
+
+> `(response: { results: Array<SearchResponse<THit> | SearchForFacetValuesResponse>, hits: MaybeArray<Hit<THit>[]>, facetHits: MaybeArray<FacetHit[]> }) => MaybeArray<Hit<THit>[] | FacetHit[]>`
+
+The function to transform the Algolia response before passing it to the Autocomplete state. You have access to the full Algolia results, as well as the pre-compiled hits and facet hits. This is useful to manipulate the hits, or store data from the results in the [context](context).
+
+This is the default implementation:
```

Comment:
That's the default implementation for `getAlgoliaResults`, not `getAlgoliaFacets`?

Reply:
Yes, hasty copy/paste. Fixed in 5b67287 and 102063d.